README.md

```diff
@@ -401,22 +493,38 @@ and scheduler algroithms. With the tuned cpu-partitioning starting with version
 available to us, we have to do this using the isolcpus option on the kernel
 command line. This can be done as follows:
 
+__For Single NUMA node system__
 ```
 sed -i -e 's/GRUB_CMDLINE_LINUX="/GRUB_CMDLINE_LINUX="isolcpus=1-13,15-27 /'  /etc/default/grub
 grub2-mkconfig -o /boot/grub2/grub.cfg
 ```
 
+__For Dual NUMA node system__
+```
+sed -i -e 's/GRUB_CMDLINE_LINUX="/GRUB_CMDLINE_LINUX="isolcpus=2-26,30-54 /'  /etc/default/grub
+grub2-mkconfig -o /boot/grub2/grub.cfg
+```
+
 
 Now it's time to reboot the machine to active the isolated cores, and use the
 configured 1G huge pages:
 
+__For Single NUMA node system__
 ```
 # reboot
 ...
 # cat /proc/cmdline
 BOOT_IMAGE=/vmlinuz-3.10.0-693.1.1.el7.x86_64 root=/dev/mapper/rhel_wsfd--netdev67-root ro default_hugepagesz=1G hugepagesz=1G hugepages=4 crashkernel=auto rd.lvm.lv=rhel_wsfd-netdev67/root rd.lvm.lv=rhel_wsfd-netdev67/swap console=ttyS1,115200 nohz=on nohz_full=1-13,15-27 rcu_nocbs=1-13,15-27 tuned.non_isolcpus=00004001 intel_pstate=disable nosoftlockup
 ```
 
+__For Dual NUMA node system__
+```
+# reboot
+...
+# cat /proc/cmdline
+BOOT_IMAGE=/vmlinuz-3.10.0-693.1.1.el7.x86_64 root=/dev/mapper/rhel_wsfd--netdev67-root ro default_hugepagesz=1G hugepagesz=1G hugepages=4 crashkernel=auto rd.lvm.lv=rhel_wsfd-netdev67/root rd.lvm.lv=rhel_wsfd-netdev67/swap console=ttyS1,115200 nohz=on nohz_full=2-26,30-54 rcu_nocbs=2-26,30-54 tuned.non_isolcpus=00004001 intel_pstate=disable nosoftlockup
```

Comment:
Guess changed this by hand, as the tuned.non_isolcpus is not correct, you might want to get this from the real machine.